ui/packages/shared/profile/src/MetricsGraph/index.tsx

```diff
@@ -58,6 +58,7 @@ export interface HighlightedSeries {
 interface Series {
   metric: Label[];
   values: number[][];
+  id: string;
```

Comment:
In this instance, we should probably call it `labelset`. 
```suggestion
  labelset: string;
```